com.avaloq.tools.ddk.check.core/META-INF/MANIFEST.MF

```diff
@@ -47,5 +47,6 @@ Export-Package: com.avaloq.tools.ddk.check,
  com.avaloq.tools.ddk.check.util,
  com.avaloq.tools.ddk.check.validation
 Import-Package: org.apache.logging.log4j,
- org.eclipse.jdt.internal.ui.text
+ org.eclipse.jdt.internal.ui.text,
+ org.apache.log4j

```

Comment:
Shouldn't we be using the new loggers everywhere now?

Reply:
The Xtext generator puts those back. I can remove them by hand if you like, but regenerating will put them back. Do you think it is worth it?